### ISSUE TITLE
tc-admin fails to install: error: taskcluster 68.0.0 is installed but taskcluster<67.1,>=44.0.0 is required by {'tc-admin'}

### ISSUE THREAD

**Issue by Archaeopteryx** [taskcluster/tc-admin#263]
When I try to install github.com/mozilla-releng/fxci-config in a fresh venv with `python setup.py develop`, it fails with the message:
`error: taskcluster 68.0.0 is installed but taskcluster<67.1,>=44.0.0 is required by {'tc-admin'}`

jcristau pointed to the the increase of the max version in 4dc73e385e499126e1644538347c2c99b36e05e6 as likely culprit.

**Comment by lotas** [comment]
Yeah, it looks like dependency constraint was upgraded by renovate bot just recently.
I think we can also just remove the upper constraint on the version at all, since taskcluster releases are not introducing breaking changes that often (at least for the tc-admin functionality) and pinning versions on the client sides should be preferred
